### PR TITLE
direnv(fix): Recommend activating the venv before running make targets

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
This was a nit of mine for a while :)

File: .envrc [hunk 1]
```diff
@@ -85,12 +85,12 @@ It's recommended to use pyenv - please refer to https://docs.sentry.io/developme
 }
 
 advice_install_sentry() {
-    info "To install sentry, please type: make install-py-dev"
+    info "To install sentry, please type: source .venv/bin/activate && make install-py-dev"
```
Comment (EvanPurkhiser) [inline .envrc:88]: how about we add a function like

```sh
advice_activate_venv() {
	if [ -z ${PYTHON_VENV} ]; then
		info "Activate your virtualenv with: source .venv/bin/activate"
		return 1
	fi
	return 0
}

advice_install_sentry() {
    advice_activate_venv() && info "To install sentry, please type: make install-py-dev"
}
```

My bash probably isn't completely right here, but you get the idea

**Comment by billyvg** [comment]
If you don't have venv activated, doesn't that mean you don't have direnv installed or `allow`'d?

**Comment by armenzg** [comment]
I can't remember how I got myself into such state (probably testing something with pre-commit).
I will open a new PR once I hit the issue again.

**Pull request closed without merging**